packages/docs/package.json

```diff
@@ -30,8 +31,6 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.7.0",
-    "concurrently": "^4.1.1",
-    "http-server": "^0.11.1",
-    "nodemon": "^1.19.1"
+    "openzeppelin-docs-preview": "github:OpenZeppelin/docs-preview-script"
```

Comment:
I see you still refuse to publish this as a package :-P
Still, if it works, fine for me!